swarm.cabal

```diff
@@ -117,6 +117,9 @@ common ghc2021-extensions
     TypeApplications
     TypeOperators
 
+-- Allow Cabal to recognize swarm as a library.
```

Comment:
It would be nice to add a comment here with a link to the Cabal issue you found that explains why this is a necessary workaround for now.